src/app/active-mode/ActiveMode.scss

```diff
@@ -0,0 +1,207 @@
+@import '../variables.scss';
+
+.activity-column {
```

Comment:
Can we use CSS modules?

Reply:
for sure. i'll push an update tonight to remove most of the sections in this file.